scarf/merge.py

```diff
@@ -74,26 +104,35 @@ def __init__(
         prepend_text: Optional[str] = "orig",
         reset_cell_filter: bool = True,
         seed: Optional[int] = 42,
+        feat_name_ids_same: Optional[bool] = False,
```

Comment:
I'd love if this was automatically detected so we don't have to pass in an extra flag.